src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -482,7 +486,8 @@
     (build-category-map [:auto-complete/complete
                          :auto-complete/prev
                          :auto-complete/next
-                         :auto-complete/shift-complete])
+                         :auto-complete/shift-complete
+                         :auto-complete/open-link])
```

Comment:
I would've preferred to have only put this on the search component but there did not seem to be an easy way to compose or add mixins to `ui/auto-complete`. Probably something we want to look into later. Looks like shift-complete also ran into this as it is search specific